local-queue.js

```diff
@@ -0,0 +1,73 @@
+const { weightSrvRecords } = require("ioredis/built/cluster/util")
+
+const queues = {}
```

Comment:
I think a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) is more suitable for this.

Reply:
True